verification/curator-service/api/src/geocoding/mapbox.ts

```diff
@@ -0,0 +1,71 @@
+import Geocoding, {
+    GeocodeFeature,
+    GeocodeMode,
+    GeocodeResponse,
+    GeocodeService,
+} from '@mapbox/mapbox-sdk/services/geocoding';
+
+import { GeocodeResult } from './geocoder';
+import { MapiResponse } from '@mapbox/mapbox-sdk/lib/classes/mapi-response';
+
+function getFeatureTypeFromContext(
```

Comment:
Could you add a comment to this function? I'm not following what it does.

And is there a reason it's outside of the class if it isn't exported?

Reply:
Added a comment.
It has no need for anything in the class, I could make it a static method but I see no real benefits in doing so.